Source/santa_driver/BUILD

```diff
@@ -121,11 +121,10 @@ function sigint() {
   sudo kextunload -b com.google.santa-driver >/dev/null
   exit 1
 }
-unzip -o $${BUILD_WORKSPACE_DIRECTORY}/bazel-bin/Source/santa-driver/santa-driver.zip >/dev/null
+unzip -o $${BUILD_WORKSPACE_DIRECTORY}/bazel-bin/Source/santa_driver/santa-driver.zip >/dev/null
 echo "Launching Kernel Tests as root. You may be prompted for your sudo password."
 trap sigint INT
-sudo $${BUILD_WORKSPACE_DIRECTORY}/bazel-bin/Source/santa-driver/kernel_tests_bin
-echo "Tests complete, unloading driver."
-sudo kextunload -b com.google.santa-driver >/dev/null
```

Comment:
I had this as a fallback in case the tests crash or the user cancels out of them because they've hung.

Reply:
Ack